cli/azd/internal/telemetry/resource/ci.go

```diff
@@ -80,3 +80,8 @@ func execEnvForCi() string {
 
 	return ""
 }
+
+// IsRunningOnCI returns true if the current process is running on a CI/CD provider.
+func IsRunningOnCI() bool {
+	return execEnvForCi() != ""
+}
```

Comment:
I'd like if we could promote the `IsRunningOnCI` to a new package outside of telemetry.
Looking at `execEnvForCi`, is there anything it requires from telemetry which justifies having it as part of it?

Reply:
Explained in the other thread. It's fairly decoupled right now.